host-bin/startandroidstudio

```diff
@@ -13,5 +13,5 @@ Wraps enter-chroot to launch Android Studio in an xiwi window.
 By default, it will log into the primary user on the first chroot found.
 Options are directly passed to enter-chroot; run enter-chroot to list them."
 
-eval "exec sh -e "`dirname "\`readlink -f "$0"\`"`/enter-chroot" -t android-studio "$@" "" \
-  exec xiwi -b /opt/google/android/studio/bin/studio.sh"
+exec sh -e "`dirname "\`readlink -f "$0"\`"`/enter-chroot" -t android-studio "$@" "" \
+  exec xiwi -b /opt/google/android/studio/bin/studio.sh
```

Comment:
actually, I don't think this is quite correct.  `xiwi` doesn't take -b, for example.  Look at `startxiwi` for reference.